PyDynamic/uncertainty/validate_FIRuncFilter.py

```diff
@@ -0,0 +1,62 @@
+import scipy.signal as scs
+import numpy as np
+import matplotlib.pyplot as plt
+
+import sys
+sys.path.append(".")
+
+from PyDynamic.uncertainty.propagate_filter import FIRuncFilter
```

Comment:
We should rather use relative imports for allowing PyDynamic to be used as a submodule.

Reply:
This does not apply, as this is an example not being packaged.

Reply:
gone